packages/core/src/components/card-list/card-list.tsx

```diff
@@ -91,6 +94,12 @@ export class CardList {
    */
   @Prop() suppressOverflowHandling = false;
 
+  /**
+   * @since 2.2.0
+   * Hide the show all button
```

Comment:
```suggestion
   * Hide the show all button
   * 
   * @since 2.2.0
```

_Apply suggestion not working here_